docs/contributing.rst

```diff
@@ -69,9 +69,9 @@ and instead, ``git rebase`` your branch.
 Implementing New Features
 -------------------------
 
-If you are writing a new feature, please ensure you write appropriate Pytest test cases and place them under ``tests/``.
+If you are writing a new feature, please ensure you write appropriate Pytest test-cases and place them under ``tests/``.
```

Comment:
not sure test cases needs to be hyphenated

Reply:
You are right, I will change it.